server/data/bookingClient.ts

```diff
@@ -111,6 +113,15 @@ export default class BookingClient {
     return response as Nonarrival
   }
 
+  async createTurnaround(premisesId: string, bookingId: string, turnaround: NewTurnaround): Promise<Turnaround> {
```

Comment:
in the API, how does this work? Does the booking have a default turnaround time that is being overridden here? I'm just thinking that 'create' seems like an odd verb to use when there is already an (expected) turnaround time for the booking.

Reply:
Yeah, it feels a bit weird, but I wanted to follow the APIs lead, up until the point we're actually showing anything to the user

From the APIs PoV, we are creating a new turnaround record, and it's the latest record that gets used for any conflict calculations. That's why the HTTP verb changed from `PUT` to `POST` and the type changed from `UpdateTurnaround` to `NewTurnaround`, which corresponds to new/create rather than edit/update

Reply:
ok thanks, makes sense 👍